### ISSUE TITLE
Adding a few more alignment widgets

### ISSUE THREAD

**Issue by JustinBraben** [rockorager/libvaxis#95]
Been working on a project using libvaxis and made some functions to make child windows aligned to each corner of the window.

bottomRight sample:
```zig
pub fn bottomRight(parent: Window, cols: usize, rows: usize) Window {
    const y_off = parent.height -| rows;
    const x_off = parent.width -| cols;
    return parent.child(.{
        .x_off = x_off, 
        .y_off = y_off, 
        .width = .{ .limit = cols }, 
        .height = .{ .limit = rows }
    });
}
```

Was wondering if I could make a PR adding them to the alignment widget? Optionally I can also create an example that uses them.

**Comment by rockorager** [comment]
A PR for this would be great. No need for an example though, it's usage is
pretty obvious.

Thanks!

-- 
Tim

**Comment by rockorager** [comment]
Fixed in 7df8ca6
